backend/models/Proposal.js

```diff
@@ -65,6 +67,35 @@ const proposalSchema = new Schema({
     ],
   },
 
+  // proposalState: {
+  //   type: String,
+  //   enum: [
+  //     "Accepted",
+  //     "Rejected",
+  //     "Granted",
+  //     "Funded",
+  //     "Not Granted",
+  //     "Down Voted",
+  //     "Withdrawn",
+  //     "Pending",
+  //   ],
+  //   default: "Accepted",
+  // },
+
+  // proposalStanding: {
```

Comment:
Commented out intentionally? Looking for logic/explanation.
<3

Reply:
Yes, in case we need them in the future. For now, we get these from airtable.